src/languages/es.ts

```diff
@@ -649,8 +649,8 @@ export default {
         nextStep: 'Pasos Siguientes',
         finished: 'Finalizado',
         submitAmount: ({amount}: RequestAmountParams) => `solicitar ${amount}`,
-        submittedAmount: ({formattedAmount, comment}: RequestedAmountMessageParams) => `solicité ${formattedAmount}${comment ? ` para ${comment}` : ''}`,
-        trackedAmount: ({formattedAmount, comment}: RequestedAmountMessageParams) => `seguimiento ${formattedAmount}${comment ? ` para ${comment}` : ''}`,
+        submittedAmount: ({formattedAmount, comment}: RequestedAmountMessageParams) => `solicitó ${formattedAmount}${comment ? ` para ${comment}` : ''}`,
```

Comment:
What do you mean?

Reply:
Yeah, it is because using submitted sounds so weird in this case. Like it depends on the word what comes next. I'd say leave it this way and I can correct the copy in the future if I find it odd since I use the app in Spanish.